R/server.R

```diff
@@ -68,14 +47,11 @@ VitessceConfigServer <- R6::R6Class("VitessceConfigServer",
     },
     #' @description
     #' Set up the server routes.
-    #' @param routes A list of route definition objects
-    #' (`VitessceConfigServerCallbackRoute` or `VitessceConfigServerStaticRoute`).
+    #' @param routes A list of `VitessceConfigServerStaticRoute` objects.
     create_routes = function(routes) {
       used_paths <- list()
       for(route in routes) {
-        if(class(route)[1] == "VitessceConfigServerCallbackRoute") {
-          private$server <- plumber::pr_get(private$server, route$path, route$callback)
-        } else {
+        if(class(route)[1] == "VitessceConfigServerStaticRoute") {
```

Comment:
This is the only `if` statement now.  Should we just get rid of it?

Reply:
Yes